tests/BenchmarkDotNet.IntegrationTests/ArgumentsTests.cs

```diff
@@ -636,10 +636,27 @@ public void Test(SomeEnum defined, SomeEnum undefined, SomeEnum undefinedNegativ
             }
         }
 
-        private void CanExecute<T>(IToolchain toolchain)
+        [Theory, MemberData(nameof(GetToolchains))]
+        public void VeryLongStringsAreSupported(IToolchain toolchain) => CanExecute<WithVeryLongString>(toolchain);
+
+        public class WithVeryLongString
         {
-            var config = CreateSimpleConfig(job: Job.Dry.With(toolchain));
-            CanExecute<T>(config);
+            private readonly string LongString = new string('a', 200_000);
+
+            public IEnumerable<object> Arguments()
+            {
+                yield return LongString;
```

Comment:
Let's add a second argument here with the following value:
```cs
private readonly string LongString2 = new string('a', 200_000 - 1) + "b";
```

Reply:
good idea! #done